tet.rs/src/tetris/board/my_board.rs

```diff
@@ -0,0 +1,382 @@
+use bit_vec::BitVec;
+use nanorand::{WyRand, RNG};
+
+use crate::tetris::board::piece::piece_matrix::{MatrixT, PieceMatrix};
+use crate::tetris::board::piece::piece_position::{PiecePosition, PositionT};
+use crate::tetris::board::piece::{
+    helper::Helper, piece_representation::PieceRepresentation, piece_rotation::PieceRotation,
+    piece_type::PieceType,
+};
```

Comment:
A quick fix would be:
```suggestion
use super::piece::piece_matrix::{MatrixT, PieceMatrix};
use super::piece::piece_position::{PiecePosition, PositionT};
use super::piece::{
    helper::Helper, piece_representation::PieceRepresentation, piece_rotation::PieceRotation,
    piece_type::PieceType,
};
```
I think you are better off reexporting just what you need in piece/mod.rs instead of looking for individual parts deep in that module.

Reply:
Yes imports were a pain and I sort of stuck to the first solution that compiled. Thanks.